components/pedigree/resources/src/main/resources/pedigree/pedigree.js

```diff
@@ -100,8 +102,9 @@ define([
             //  drawNodeShadows:              {true|false}   - display small shadow under node graphic; default: "true"
             //  disabledFields:               [array]        - list of node-menu fields disabled for this installation
             //  replaceIdWithExternalID:      {true|false}   - when true, patient links display external ID as the link text (instead of PT ids)
-            //  displayCancerLabels:          {true|false}   - display labels for each afecting cancer; default: "true"
+            //  displayCancerLabels:          {true|false}   - display labels for each affecting cancer; default: "true"
             //  lineStyle:                    {"thin"|"regular"|"bold"} - controls the thickness of all lines in pedigree
+            //  studies:                      [array]        - array of all available studies
```

Comment:
need to expand comment to say something like `array of available study objects in the following format: {"id": ..., "name": ..., "description": ...}`  to make it clear what kind of data is expected there